### DISCUSSION TITLE
my shuffledns is not working

### DISCUSSION THREAD

**Discussion by Huzaifa-cloud** [projectdiscovery/shuffledns#412] [General]
i am facing this problem
[FTL] Program exiting: resolver file doesn't exists
 i have also changed my resolver and wordlist but issue occurs please help me to solve it

**Comment by dogancanbakir** [comment]
Please provide all the details necessary for us to reproduce the issue. Thank you!

**Comment by Huzaifa-cloud** [comment]
what kind of details you want to resolve it

**Comment by dogancanbakir** [comment]
```
<!-- 
1. Please search to see if an issue already exists for the bug you encountered.
2. For support requests, FAQs or "How to" questions, please use the GitHub Discussions section instead - https://github.com/projectdiscovery/shuffledns/discussions or
3. Join our discord server at https://discord.gg/projectdiscovery and post the question on the #shuffledns channel.
-->

<!-- ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION. -->

### shuffledns version:
<!-- You can find current version of shuffledns with "shuffledns -version" -->
<!-- We only accept issues that are reproducible on the latest version of shuffledns. -->
<!-- You can find the latest version of project at https://github.com/projectdiscovery/shuffledns/releases/ -->

### Current Behavior:
<!-- A concise description of what you're experiencing. -->

### Expected Behavior:
<!-- A concise description of what you expected to happen. -->

### Steps To Reproduce:
<!--
Example: steps to reproduce the behavior:
1. Run 'shuffledns ..'
2. See error...
-->


### Anything else:
<!-- Links? References? Screnshots? Anything that will give us more context about the issue that you are encountering! -->
```

**Comment by Huzaifa-cloud** [comment]
![Screenshot_2024-09-06_07_34_45](https://github.com/user-attachments/assets/66533281-2f32-4b58-b580-c788892b5982)

**Comment by Huzaifa-cloud** [comment]
[FTL] Program exiting: execution mode not specified 
what is the meaning of this error and it it is happening

**Reply by Mzack9999** [reply]
As you can see in the command life error you need to ensure that the file resolver file exists in the path you are specifying.
Additionally you need to define the execution mode (bruteforce, resolve, etc) with for example `-mode bruteforce`.

When opening an issue please provide all the info and steps to reproduce the problem. We can't fix what we are not able to reproduce.

**Reply by captain-arslan** [reply]
thank u my problem is solved
thx my problem has been solved
